src/components/Menu.vue

```diff
@@ -0,0 +1,96 @@
+<script>import algoliaComponent from '../component';
+  import { FACET_TREE } from '../store';
+
+  export default {
+    mixins: [algoliaComponent],
+
+    props: {
+      attribute: {
+        type: String,
+      },
+      limit: {
+        type: Number,
+        default: 10,
+      },
+      sortBy: {
+        default() {
+          return ['isRefined:desc', 'count:desc', 'name:asc'];
+        },
+      },
+    },
+
+    computed: {
+      facetValues() {
+        const { data = [] } = this.searchStore.getFacetValues(
+          this.attribute,
+          this.sortBy
+        );
+
+        return data;
+      },
+    },
+
+    methods: {
+      handleClick(event, path) {
+        event.preventDefault();
+        this.searchStore.toggleFacetRefinement(this.attribute, path);
+      },
+    },
+
+    data() {
+      return {
+        blockClassName: 'ais-menu',
+      };
+    },
+
+    created() {
+      this.searchStore.addFacet(
+        {
+          name: this.attribute,
+          attributes: [this.attribute],
+        },
+        FACET_TREE
+      );
+    },
+
+    destroyed() {
+      this.searchStore.removeFacet(this.attribute);
+    },
+
+    render(h) {
+      if (this.show === false) return undefined;
+
+      const children = [];
+      if (this.$slots.header) children.push(this.$slots.header);
+
+      children.push(
+        this.facetValues.map(({ name, path, count, isRefined }) =>
+          h(
+            'div',
+            {
+              class: [
+                this.bem('item'),
+                isRefined ? this.bem('item', 'active') : '',
```

Comment:
`isRefined ? this.bem('item', 'active') : this.bem('item')`

By default, `this.bem('item', 'active')` will output `ais-menu__item ais-menu__item--active`. To avoid duplicate you can do the above suggestion, or pass `false` as the last parameter to avoid outputting element class: `this.bem('item', 'active', false)`